portfolio/src/main/webapp/hobbies/script.js

```diff
@@ -0,0 +1,13 @@
+function addRandomSong() {
+  /*
```

Comment:
I'm guessing you intended on deleting the comment and implementing later. What we usually do to remind ourselves/notify other collaborators that we intend on working on something later is leave a comment formatted like "TODO(frankyg): Implement this."